webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -22,7 +22,8 @@
     v-draggable="enabled"
     disable-pull-to-refresh
     right
-    fixed>
+    fixed
+    @closed="resetEditor">
```

Comment:
I think no need of this since the editor will be deleted once the drawer gets closed, see it here https://github.com/Meeds-io/social/blob/69cee6529fe63b78d6f74264af1783e8831b2e56/webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue#L118